### DISCUSSION TITLE
senting event texts to serial port

### DISCUSSION THREAD

**Discussion by zajc3w** [EDCD/EDDI#2660] [EDDI Standalone Q&A]
I just discovered EDDi searching for ways to communicate game events to external device(meachnical, arduino driven cockpit).
Idea is to have backlit status labels for gear/cargo scoop etc. All I need is output "landing gear deplyed=true" to serial port etc.
i could probably hook up my own serial driver to EDDI API if thats exposed.

**Comment by Tkael** [comment]
Unfortunately EDDI does not offer a general purpose API at this time.
